docs/api/validation-chain.md

```diff
@@ -387,6 +387,94 @@ body('json_string').optional().isLength({ max: 100 }).isJSON().
 
 :::
 
+### `.hide()`
+
+```ts
+hide(hiddenValue?: string): ValidationChain
+```
+
+Hide field's value in the returning errors of validationResult() method.
+If the value is confidential information (such as api key),
+please call this method to prevent exposing it.
+If hiddenValue is not given, value property itself is removed.
+
+| Name          | Description                               |
+| ------------- | ----------------------------------------- |
+| `hiddenValue` | String to be replaced with field's value. |
+
+:::info
```

Comment:
I'm not sure about the examples in this block.
Maybe we can get away with something like this?

```ts
// Omits the value in the errors
query('api_key').custom(isValidKey).hide(); 

// Replaces the value in the errors with '*****'
query('api_key').custom(isValidKey).hide('*****'); 
```